Spotify.py

```diff
@@ -102,31 +102,33 @@ def handle_message(channel, user, message):
                     return "Already used volume up in the last {} minutes. Try again later".format(config.VOLUME_TIME)
 
             if command in COMMAND_DELETE:
-                try:
-                    params = int(params)
-                    if 5 > params > 1 and Spotify.check_user(user):
-                        Spotify.execute_command("del {}").format(params)
-                        USERS[user] = datetime.datetime.now() + datetime.timedelta(minutes=config.WAIT_TIME)
-                    else:
-                        return "Already skipped or deleted a song from the queue in the last {} minutes. " \
-                               "Try again later".format(config.WAIT_TIME)
-                except ValueError:
-                    return "Please choose a number between 1 - {}".format(str(config.QUEUE_LENGTH))
+                if params is not None:
+                    try:
+                        params = int(params)
+                        if 5 > params > 1 and Spotify.check_user(user):
+                            Spotify.execute_command("del {}").format(params)
+                            USERS[user] = datetime.datetime.now() + datetime.timedelta(minutes=config.WAIT_TIME)
+                        else:
+                            return "Already skipped or deleted a song from the queue in the last {} minutes. " \
+                                   "Try again later".format(config.WAIT_TIME)
+                    except ValueError:
+                        return "Please choose a number between 1 - {}".format(str(config.QUEUE_LENGTH))
 
             if command in COMMAND_RANDOM:
-                param_first, param_second = params.split(None, 1)
-                try:
-                    param_first = int(param_first)
-
-                    if param_second.__eq__(''):
-                        return random.randint(1, int(param_first))
-                    elif param_second.isdigit():
-                        return random.randint(param_first, int(param_second))
-                    else:
+                if params is not None:
```

Comment:
What happens if params is None?

Reply:
Changed if split or conversion fails, returns random number 1-10